isa_helper.c

```diff
@@ -4,9 +4,7 @@
 uint32_t rotate_right(uint32_t shiftee, uint8_t shifter)
 {
     return (shiftee << (32 - shifter)) | (shiftee >> (shifter));
-}
-
-uint8_t get_bit(uint32_t from, uint8_t bitid)
+} uint8_t get_bit(uint32_t from, uint8_t bitid)
```

Comment:
I didn't do anything, I'm even using your shitty vimrc additions.

On 05-Oct-2016 10:52 PM, "Jai Luthra" <notifications@github.com> wrote:

> In isa_helper.c
> <https://github.com/static-code-generators/armsimc/pull/11#discussion_r82021919>
> :
>
> > @@ -4,9 +4,7 @@
> >  uint32_t rotate_right(uint32_t shiftee, uint8_t shifter)
> >  {
> >      return (shiftee << (32 - shifter)) | (shiftee >> (shifter));
> > -}
> > -
> > -uint8_t get_bit(uint32_t from, uint8_t bitid)
> > +} uint8_t get_bit(uint32_t from, uint8_t bitid)
>
> pls dont play around with whitespace, revert this to original, and i know
> this is @parthmittal <https://github.com/parthmittal> 's fault, but you
> fix it pls
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/static-code-generators/armsimc/pull/11/files/3e27c6b57a33cb236882a230b21f5cd3e67dc28f#r82021919>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AND8vHfeliNMmKMtPBs7BZQNswmDVe4Gks5qw9zcgaJpZM4KO8qO>
> .
>